src/utils/gas.ts

```diff
@@ -24,15 +24,19 @@ export type GasPricing = {
 }
 
 // Implemented from https://eips.ethereum.org/EIPS/eip-1559
-export function calculateNextBlockFee(parent: Block | BlockWithTransactions): BigNumber {
+export function calculateNextBlockFee(network: string, parent: Block | BlockWithTransactions): BigNumber {
   const zero: BigNumber = BigNumber.from('0')
   if (parent.baseFeePerGas === undefined) {
     return zero
   }
 
   const one: BigNumber = BigNumber.from('1')
   const elasticityMultiplier: BigNumber = BigNumber.from('2')
-  const baseFeeMaxChangeDenominator: BigNumber = BigNumber.from('8')
+  let baseFeeMaxChangeDenominator: BigNumber = BigNumber.from('8')
+  if (network === 'polygon' || network === 'plygonTestnet') {
```

Comment:
@alexanderattar you misspelled `polygonTestnet`. Also, was the network value capitalized here, or was it only in the logger?